SQL/0000-00-00-schema.sql

```diff
@@ -2014,3 +2016,28 @@ CREATE TABLE `feedback_mri_comments` (
   CONSTRAINT `FK_feedback_mri_comments_2` FOREIGN KEY (`PredefinedCommentID`) REFERENCES `feedback_mri_predefined_comments` (`PredefinedCommentID`),
   CONSTRAINT `FK_feedback_mri_comments_3` FOREIGN KEY (`FileID`) REFERENCES `files` (`FileID`)
 ) ENGINE=InnoDB DEFAULT CHARSET=utf8;
+
+CREATE TABLE `visit` (
+  `VisitID` int(10) unsigned NOT NULL auto_increment,
+  `VisitLabel` varchar(255) NOT NULL,
+  `VisitLegacyLabel` varchar(255) DEFAULT NULL,
```

Comment:
I don't understand what a "VisitLegacyLabel" is.

Reply:
After discussion with Rida and Melanie (separately), this table will contain a `name` column (what we currently call visit_label) and an autoincrement `id` column for convenience. The intention was to have a third column for displayed labels. since this functionnality do not exists yet, we will stick with 2 column and add a third when the time comes (if ever.)

Reply:
The short version: I had originally proposed the legacy label in my initial changes over a year ago with the intent of slowly deprecating the underscore separated visit labels in favour of a front end label and an ID. However given the importance of the visitlabels in their current form for the file and imaging uploads this change might be premature.

Instead i propose following our newly established standards and creating a visitid, name and label fields as follows:
- `VisitID=1`
- `Name = the_first_visit`
-  `Label = The First Visit`

Tgis allows us to maintain current functionality while adding room for improvement and following the standards